### ISSUE TITLE
Common Components - Club Listing Card

### ISSUE THREAD

**Issue by xavjones8** [NCSU-App-Development-Club/Marigold-Android#13]
# **Description**:

Design hardcoded club listing cards. Create a new composable files for each Figma component in a new "common" package.

# **Accessibility Requirements**:
Text and content should be readable at maximum 200% screen zoom.
The accessibility label should announce the card as a button, and read off all the text in the card.

# **Links and Resources**:

Draft designs can be found here under the **************Xavier************** tab, but are not finalized

https://www.figma.com/file/raH7fqviVPnDFKILPBJ5iN/NCSU-Club-%26-Events-App?type=design&node-id=0-1&mode=design&t=vAJwOwAhwt5KSvGg-0

# **Attachments**:
![ClubListingTableRow](https://github.com/NCSU-App-Development-Club/Marigold-Android/assets/44850083/f4231cd1-d39b-471f-baef-d8d88fdea1fb)
